dynamic_models/factory.py

```diff
@@ -92,15 +83,7 @@ def make_field(self):
         return constructor(**options)
 
     def get_constructor(self):
-        return self.DATA_TYPES[self.schema.data_type]
-
-    @classmethod
-    def data_type_choices(cls):
-        return [(dt, dt) for dt in cls.get_data_types()]
-
-    @classmethod
-    def get_data_types(cls):
-        return cls.DATA_TYPES
+        return getattr(models, self.schema.class_name)
```

Comment:
I think we can add a lot more flexibility here. If we make this a fully qualified class name, e.g. `django.db.models.IntegerField`, we would be able to support custom defined field classes. 

The only change here would be to `rsplit('.', maxsplit=1)` the value into the module and the class name. You can use `importlib` to import the module and `getattr` just like you have to get the class.

Reply:
yeah right, this will definitely help flexibility.